awx/ui/test/e2e/tests/test-users-crud.js

```diff
@@ -0,0 +1,54 @@
+/* Tests for the user CRUD operations. */
+import { genID } from '../utils';
```

Comment:
This feels like an unnecessary abstraction to me. Other tests directly import the `uuid` library and use it in their tests like so:

```
const id = uuid().substr(0, 8);
```
See `e2e/tests/smoke.js` or something similar for reference.
Is there any added benefit to pulling out of context in this specific test?

Reply:
This was an initial attempt to abstract that and have the id generation on a single place. A next step would be updating the rest of the test suite to use that instead.

I sent this way to gather feedback and see if it really worth or not.